### ISSUE TITLE
TensorforceTradingStrategy looks broken 

### ISSUE THREAD

**Issue by macd2** [notadamking/tensortrade#101]
this strategy is not working and no matter what it results in a 

```   
ValueError: Cannot feed value of shape (1, 10) for Tensor 'agent/state-input:0', which has shape '(?, 20, 10)'
```

This error is related to tensorforce the shape of actions is not correctly passed

**Comment by notadamking** [comment]
I believe this is a bug in Tensorforce, as they are not feeding the correct shape when the `window_size` is > 1.

**Comment by notadamking** [comment]
Possibly solved in Tensorforce 0.5.2
